### PR TITLE
Remaining small TODOs

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

Fixes a few small remaining TODOs.

### CODE REVIEW

File: src/components/Codeblock.tsx [hunk 1]
```diff
@@ -321,13 +322,9 @@ const Codeblock: React.FC<IProps> = ({
                     totalLines - 1 > LINES_BEFORE_COLLAPSABLE && (
                       <TopBarItem onClick={() => setIsCollapsed(!isCollapsed)}>
                         {isCollapsed ? (
-                          // TODO: Implement after intl
-                          // <Translation id="show-all" />
-                          <>Show all</>
+                          <Translation id="show-all" />
```
Comment (nhsz) [inline src/components/Codeblock.tsx]: we should use the hook `t` with `common` namespace for these cases

File: src/components/Codeblock.tsx [hunk 2]
```diff
@@ -321,13 +322,9 @@ const Codeblock: React.FC<IProps> = ({
...
                         ) : (
-                          // TODO: Implement after intl
-                          // <Translation id="show-less" />
-                          <>Show less</>
+                          <Translation id="show-less" />
```
Comment (nhsz) [inline src/components/Codeblock.tsx]: we should use the hook `t` for these cases

File: src/components/Codeblock.tsx [hunk 3]
```diff
@@ -338,16 +335,12 @@ const Codeblock: React.FC<IProps> = ({
                           {!isCopied ? (
                             <>
                               <Emoji text=":clipboard:" fontSize="md" />{" "}
-                              {/* TODO: Implement after intl */}
-                              {/* <Translation id="copy" /> */}
-                              Copy
+                              <Translation id="copy" />
```
Comment (nhsz) [inline src/components/Codeblock.tsx]: we should use the hook `t` for these cases

File: src/components/Codeblock.tsx [hunk 4]
```diff
@@ -338,16 +335,12 @@ const Codeblock: React.FC<IProps> = ({
...
                             </>
                           ) : (
                             <>
                               <Emoji text=":white_check_mark:" fontSize="md" />{" "}
-                              {/* TODO: Implement after intl */}
-                              {/* <Translation id="copied" /> */}
-                              Copied
+                              <Translation id="copied" />
```
Comment (nhsz) [inline src/components/Codeblock.tsx]: we should use the hook `t` for these cases

**Review by nhsz — CHANGES_REQUESTED** [review]
Thanks, asking for a small change to keep consistency for translations

**Review by corwintines — APPROVED** [review]

**Review by nhsz — APPROVED** [review]

**Pull request merged**